tests/queries/0_stateless/01889_sql_json_functions.reference

```diff
@@ -0,0 +1,38 @@
+--JSON_VALUE--
+
+1
+1.2
+true
+"world"
+null
+
+
+
+
+--JSON_QUERY--
+[{"hello":1}]
+[1]
+[1.2]
+[true]
+["world"]
```

Comment:
Can you please also add tests when json query returns more elements in an array than just one. And also case when second argument is a column with multiple rows.